colossalai/kernel/triton/context_attention.py

```diff
@@ -0,0 +1,184 @@
+import torch
+import math
+try:
+    import triton
+    import triton.language as tl
+    HAS_TRITON = True
+except ImportError:
+    HAS_TRITON = False
```

Comment:
If `import triton` meets error, it seems that the framework can not work.
The error handling seems meaningless.

Reply:
This is set for not hurting the other code.